core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -1107,7 +1108,8 @@ private static short encodeDeliverMode(DeliverOrder deliverOrder) {
          case PER_SENDER:
             return Message.Flag.NO_TOTAL_ORDER.value();
          case NONE:
-            return (short) (Message.Flag.OOB.value() | Message.Flag.NO_TOTAL_ORDER.value());
+            return (short) (Message.Flag.OOB.value() | Message.Flag.NO_TOTAL_ORDER.value() |
+                            Message.Flag.DONT_BUNDLE.value());
```

Comment:
why setting DONT_BUNDLE? this will bypass the bundler making no point to introduce a new bunlder.

Reply:
`DONT_BUNDLE` has been ignored by the bundler for some time now, it's only used on the target to deliver each message in a separate thread or as a batch on a single thread.

Some of our commands block, especially in tests, and if multiple commands are delivered in a batch then one blocking command will block all the other commands remaining in the batch. We can try removing `DONT_BUNDLE` after all the commands are really non-blocking.